src/commands/deploy.ts

```diff
@@ -70,42 +60,16 @@ export abstract class DeployCommand extends Command {
 
 export default class Deploy extends DeployCommand {
   auth_required(): boolean {
-    const { flags } = this.parse(Deploy);
-    return !flags.local;
+    return true;
   }
 
-  static description = 'Create a deploy job on Architect Cloud or run stacks locally';
+  static description = 'Create a deploy job on Architect Cloud';
 
   static flags = {
     ...DeployCommand.flags,
     ...AccountUtils.flags,
     ...EnvironmentUtils.flags,
 
-    local: flags.boolean({
```

Comment:
Could we support --local for a period of time? Add a deprecated label `${Command.DEPRECATED} ..`.

It says its an anti-pattern, but it'd be temporary.

```ts
if (flags.local) {
  // https://oclif.io/docs/running_programmatically#calling-commands-directly
  await DevCommand.run([...])
}
```

Reply:
This makes sense and is easy enough. If we do this though, I would prefer to add an update by timeframe. This would also let us put a notice for when users need to upgrade their usage of the command by. For this specific instance I would say a month is more than adequate.